.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "ced"]
 	path = ced
 	url = https://github.com/performous/compact_enc_det
+[submodule "3rdparty/aubio"]
+	path = 3rdparty/aubio
+	url = git@github.com:performous/aubio.git
```

Comment:
Change this to use https url instead of ssh as the above ced module. Else change ced to use ssh

Reply:
Can I take care of this later? That one branch I always talk about also moved ced to the 3rdparty folder (the idea is to consolidate all submodules there) so I can fix that in there also?

Reply:
@Lord-Kamina next time link the new issue while resolving the convo thanks! 👍